public/pages/RollupDetails/containers/RollupDetails/RollupDetails.tsx

```diff
@@ -331,11 +331,16 @@ export default class RollupDetails extends Component<RollupDetailsProps, RollupD
       isDeleteModalVisible,
     } = this.state;
 
-    let scheduleText = continuousJob ? "Continuous, " : "Not continuous, ";
-    if (continuousDefinition == "fixed") {
-      scheduleText += "every " + interval + " " + parseTimeunit(intervalTimeunit);
+    let scheduleText = "";
+    if (continuousJob == "yes") {
+      scheduleText += "Continuous, ";
+      if (continuousDefinition == "fixed") {
+        scheduleText += "every " + interval + " " + parseTimeunit(intervalTimeunit);
+      } else {
+        scheduleText += "defined by cron expression: " + cronExpression;
+      }
     } else {
-      scheduleText += "defined by cron expression: " + cronExpression;
+      scheduleText += "Not continuous";
```

Comment:
Are these same logic as the other place in ScheduleRolesAndNotifications.tsx? Can extract this out.